automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/analysis/MethodConfigSpec.scala

```diff
@@ -0,0 +1,68 @@
+package org.broadinstitute.dsde.firecloud.test.analysis
+
+import org.broadinstitute.dsde.firecloud.fixture.UserFixtures
+import org.broadinstitute.dsde.firecloud.page.workspaces.WorkspaceDataPage
+import org.broadinstitute.dsde.firecloud.page.workspaces.methodconfigs.WorkspaceMethodConfigDetailsPage
+import org.broadinstitute.dsde.workbench.auth.AuthToken
+import org.broadinstitute.dsde.workbench.config.UserPool
+import org.broadinstitute.dsde.workbench.fixture._
+import org.broadinstitute.dsde.workbench.service.test.WebBrowserSpec
+import org.scalatest.{FreeSpec, Matchers}
+
+class MethodConfigSpec extends FreeSpec with Matchers with WebBrowserSpec with WorkspaceFixtures with UserFixtures with MethodFixtures with BillingFixtures {
+
+  "input/output auto-suggest" - {
+    "stays current with selected root entity type" in {
+      val user = UserPool.chooseProjectOwner
+      implicit val authToken: AuthToken = user.makeAuthToken()
+      withCleanBillingProject(user) { projectName =>
+        withWorkspace(projectName, "MethodConfigSpec", attributes = Some(Map("foo" -> "bar"))) { workspaceName =>
+          val configName = s"${SimpleMethodConfig.configName}_$workspaceName"
+          api.methodConfigurations.createMethodConfigInWorkspace(
+            projectName, workspaceName,
+            MethodData.SimpleMethod,
+            projectName, configName,
+            SimpleMethodConfig.snapshotId,
+            Map.empty, Map.empty,
+            "sample")
+
+          withWebDriver { implicit driver =>
+            withSignIn(user) { _ =>
+              val configPage = new WorkspaceMethodConfigDetailsPage(projectName, workspaceName, projectName, configName).open
+              configPage.isEditing shouldBe false
+
+              configPage.toggleDataModel() shouldBe false
+              configPage.isEditing shouldBe true
+              configPage.clickAndReadSuggestions("test.hello.name") should contain theSameElementsAs Seq("workspace.foo")
```

Comment:
would it be useful to stick `Seq("workspace.foo")` and `Seq("workspace.foo", "this.is_a_participant")` into vals (like "workspaceAttributes" and "workspaceAndParticipantAttributes" ?)